Source/WebCore/fileapi/Blob.cpp

```diff
@@ -304,6 +304,27 @@ void Blob::arrayBuffer(Ref<DeferredPromise>&& promise)
     });
 }
 
+void Blob::bytes(Ref<DeferredPromise>&& promise)
+{
+    loadBlob(FileReaderLoader::ReadAsArrayBuffer, [promise = WTFMove(promise)](BlobLoader& blobLoader) mutable {
+        if (auto optionalErrorCode = blobLoader.errorCode()) {
+            promise->reject(Exception { *optionalErrorCode });
+            return;
+        }
+        auto arrayBuffer = blobLoader.arrayBufferResult();
+        if (!arrayBuffer) {
+            promise->reject(Exception { ExceptionCode::InvalidStateError });
+            return;
+        }
+        auto view = Uint8Array::tryCreate(WTFMove(arrayBuffer), 0, arrayBuffer->byteLength());
+        if (!view) {
+            promise->reject(Exception { ExceptionCode::InvalidStateError });
```

Comment:
InvalidStateError or OutOfMemoryError?
We should look at what we are doing for arrayBuffer.

Ditto for `if (!arrayBuffer)`